bitmap-nassarah/test/bitmap-read-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const assert = require('assert');
+
+const bitmapRead = require('../lib/bitmap-read');
+
+describe('testing whether we can read buffer of any file', function() {
+  it('should return a buffer', function(done) {
+    bitmapRead('test.txt', function(err, result){
+      console.log(err);
+      var resultString = result.toString();
+      assert.equal(resultString, '111\n', 'was not 111');
+      done();
```

Comment:
This is technically testing Node code (from the module fs) and is unnecessary. There's not much to test from this module, as you're wrapping fs code in your own function. You only want to test code that you wrote.